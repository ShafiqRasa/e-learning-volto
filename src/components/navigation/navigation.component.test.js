import { render } from '@testing-library/react';
import Navigation from './navigation.component';

describe('Test: Navigation element', () => {
  test("match the Navigation component with it's previous snapShot, nothing should be changed!", () => {
    const NavigationElement = render(<Navigation />);

    expect(NavigationElement).toMatchSnapshot();
  });
});
