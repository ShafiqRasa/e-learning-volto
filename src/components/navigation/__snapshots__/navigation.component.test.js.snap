// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test: Navigation element match the Navigation component with it's previous snapShot, nothing should be changed! 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bqGGcD YaJtT"
      >
        <section
          class="sc-eCAqax gFhruJ"
        >
          <p>
            Navigation
          </p>
        </section>
        <div
          class="sc-hKFymg eGFjGA"
        >
          <img
            alt="fraud_prevention.png"
            src="/assets/icons/fraud_prevention.png"
          />
          <p>
            Money Laundering and Fraud Prevention
          </p>
        </div>
        <div
          class="sc-hKFymg eGFjGA"
        >
          <img
            alt="it_security.png"
            src="/assets/icons/it_security.png"
          />
          <p>
            IT Secourity and Information Secourity
          </p>
        </div>
        <div
          class="sc-hKFymg eGFjGA"
        >
          <img
            alt="data_protection.png"
            src="/assets/icons/data_protection.png"
          />
          <p>
            Data Protection
          </p>
        </div>
        <div
          class="sc-hKFymg eGFjGA"
        >
          <img
            alt="wphg.png"
            src="/assets/icons/wphg.png"
          />
          <p>
            WpHG-Compliance
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bqGGcD YaJtT"
    >
      <section
        class="sc-eCAqax gFhruJ"
      >
        <p>
          Navigation
        </p>
      </section>
      <div
        class="sc-hKFymg eGFjGA"
      >
        <img
          alt="fraud_prevention.png"
          src="/assets/icons/fraud_prevention.png"
        />
        <p>
          Money Laundering and Fraud Prevention
        </p>
      </div>
      <div
        class="sc-hKFymg eGFjGA"
      >
        <img
          alt="it_security.png"
          src="/assets/icons/it_security.png"
        />
        <p>
          IT Secourity and Information Secourity
        </p>
      </div>
      <div
        class="sc-hKFymg eGFjGA"
      >
        <img
          alt="data_protection.png"
          src="/assets/icons/data_protection.png"
        />
        <p>
          Data Protection
        </p>
      </div>
      <div
        class="sc-hKFymg eGFjGA"
      >
        <img
          alt="wphg.png"
          src="/assets/icons/wphg.png"
        />
        <p>
          WpHG-Compliance
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
